src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const fullList = document.querySelectorAll('li');
+
+function sortList(list) {
+  return [...list].sort((a, b) => {
+    const salaryA = +(a.dataset.salary.slice(1).split('')
+      .filter(item => item !== ',').join(''));
```

Comment:
better to use `replace` instead of `split.filter.join`. write helper function for parsing and use it for salary a/b